src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,42 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int SIZE_OF_STORAGE = 10;
+    private final K[] keysOfStorage;
+    private final V[] valuesOfStorage;
+    private int sizeOfStorage;
```

Comment:
Variables should not have a class name
```suggestion
    private final K[] keys;
    private final V[] values;
    private int size;
```